parquet/src/test/java/org/apache/iceberg/parquet/TestDictionaryRowGroupFilter.java

```diff
@@ -18,37 +18,13 @@
  */
 package org.apache.iceberg.parquet;
 
-import static org.apache.iceberg.avro.AvroSchemaUtil.convert;
```

Comment:
these shouldn't change

Reply:
Solved in 833e9bff045287bc164e96c1fcd221fb79cd4138.